index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="style.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
+</head>
+
+<body>
+
+    <header class="navibar">
+        <p class="name">John Doe</p>
```

Comment:
- [ ] In this design, this name should be treated as a logo (if you don't want to implement a real logo) so please make sure to change the `p` tag into an `a` tag and make sure once you click it, it redirects to the main page.